app/models/alchemy/element/presenters.rb

```diff
@@ -43,7 +43,8 @@ def display_name
     #   Length of characters after the text will be cut off.
     #
     def preview_text(maxlength = 30)
-      (contents.detect(&:preview_content?) || contents.first).try(:preview_text, maxlength)
+      preview_content = contents.detect(&:preview_content?)
```

Comment:
I would like to have a public method `preview_content`

```ruby
def preview_content
  @_preview_content ||= contents.detect(&:preview_content?)
end
```

Then you don't need to pass this variable into both methods.